docs/plugin_developers_guide/02_writing_plugins.rst

```diff
@@ -392,6 +392,44 @@ The configuration items within a configuration category can each be defined as o
    * - object
      - A complex configuration type with multiple elements that may be used within *list* and *kvlist* items only, it is not possible to have *object* type items outside of a list. Object type configuration items have a set of *properties* defined, each of which is itself a configuration item. 
 
+List
+####
+
+A simple list would be defined using the JSON shown below
+
+.. code-block:: JSON
+
+ "tags" : {
+                "description" : "A set of tag names on which to operate",
+                "type" : "list",
+                "items" : "string",
+                "default" : "[ \"speed\", \"temperature\", \"voltage\" ]",
+                "order" : "4",
+                "displayName" : "Labels"
+           }
+
+The returned value for such a configuration item is merely an array of strings.
+
+.. code-block:: JSON
+
+    [ "speed", "temperature", "voltage", "current", "status" ]
+
+In the above example the user is able to supply any number of unconstrained strings, however if we wanted a user to choose from a set of fixed options they may use an enumeration as the type of the list rather than a string.
```

Comment:
We do not see much point in a simple list that can only take a subset of values with enumeration. Makes more sense with kvlist type. Though API have a support for a list with enumeration. We may exclude it from doc.

Reply:
Yes, there is a number of times when you want to ave something that is effectively choose n thing from m choices.